plugins/BEdita/API/src/Controller/ObjectsController.php

```diff
@@ -159,8 +159,7 @@ public function index()
         } else {
             // List existing entities.
             $filter = (array)$this->request->getQuery('filter') + array_filter(['query' => $this->request->getQuery('q')]);
-            $include = $this->request->getQuery('include');
-            $contain = $include ? $this->prepareInclude($include) : [];
+            $contain = $this->prepareInclude($this->request->getQuery('include'));
```

Comment:
Scrutinizer suggests some type check like:
```$this->prepareInclude((string)$this->request->getQuery('include'))```

but we should then change `prepareInclude` BadRequest test, at least.... :thinking: